mysql-common/usr/share/cont-docs/30-mysql.txt.tpl

```diff
@@ -32,36 +32,52 @@ You can also set following mount points by passing `-v /host:/container` flag to
 
 
 Usage
----------------------------------
+-----
 
 To just run the dameon and not store the database in a host directory,
 you need to execute the following command:
 
 ```
-# docker run --name={{ collection }} -d -p 3306:3306 <THIS_IMAGE>
+# docker run --name={{ collection }} -d -p 3306:3306 THIS_IMAGE
 ```
 
-If you want to set only environment variables to create a database and not store
-the database in a host directory, you need to execute the following command:
+This will run the daemon in default configuration and port 3306 will be
+exposed and mapped to host.
+
+It is recommended to use run the container with mounted data directory everytime.
+This example shows how to run the container with `/host/data` directory mounted
+and so the database will store data into this directory on host:
 
 ```
-# docker run -d --name mysql_database -e MYSQL_USER=user -e MYSQL_PASSWORD=pass -e MYSQL_DATABASE=db -p 3306:3306 <THIS_IMAGE>
+docker run -d --name mysql_database -v /host/data:/var/lib/mysql/data THIS_IMAGE
 ```
 
-This will create a container named `mysql_database` running MySQL with database
-`db` and user with credentials `user:pass`. Port 3306 will be exposed and mapped
-to host.
+This will create a container named `mysql_database` running {{ name }} daemon
+and storing data into `/host/data` on the host.
+
+For debugging purposes or just connecting to the running container, run
+`docker exec -ti mysql_database container-entrypoint` in a separate terminal.
+
+You can stop the detached container by running `docker stop mysql_database`.
+
 
-If you want your database to be persistent across container executions,
-also add a `-v /host/db/path:/var/lib/mysql/data` argument. This is going to be
-the MySQL data directory.
+Database initialization
+-----------------------
 
 If the database directory is not initialized, the container script will first
 run [`mysql_install_db`](https://dev.mysql.com/doc/refman/5.5/en/mysql-install-db.html)
 during start and setup necessary database users and passwords. After the database is
 initialized, or if it was already present, `mysqld` is executed and will run as PID 1.
 
-You can stop the detached container by running `docker stop mysql_database`.
+To pass arguments that are used for initializing the database if it is not yet
+initialized, define them as environment variables
+
+```
+docker run -d --name mysql_database -e MYSQL_USER=user -e MYSQL_PASSWORD=pass -e MYSQL_DATABASE=db THIS_IMAGE
```

Comment:
Can you remove the --name here? If you start this container and then you stop it, I think the default Docker behavior is that it will refuse to start a new one until you remove the 'mysql_database' one ;-)